### PR TITLE
Remove the override of `Throwable` in the minilib.

### PR DESCRIPTION
**Pull request by sjrd**
The only blocker was calling `Utils.isUndefined`, which we manually inline instead.

The minilib is now free of any override. It is strictly a subset of the standard java lib. Therefore, it does not need its own project anymore, and is turned into just a custom task of the library.

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -773,7 +775,7 @@ object Build {
 
   lazy val linker: MultiScalaProject = MultiScalaProject(
       id = "linker", base = file("linker/jvm")
-  ).zippedSettings("minilib", "library")(
```
Comment (gzm0) [inline project/Build.scala]: I believe this overload of `zippedSettings` is now unused and can be removed.

File: project/Build.scala [hunk 2]
```diff
@@ -1354,35 +1356,20 @@ object Build {
...
-      scalaJSExternalCompileSettings,
-  ).zippedSettings(library)(library =>
-      inConfig(Compile)(Seq(
-          mappings in packageBin := {
-            val superMappings = (mappings in packageBin).value
-            val libraryMappings = (mappings in (library, packageBin)).value
+          },
 
-            val whitelisted = libraryMappings.filter { mapping =>
+          packageMinilib := {
```
Comment (gzm0) [inline project/Build.scala]: Have you considered using `Defaults.packageTaskSettings`? (I'm not sure it'll turn out simpler, just wondering if you have looked at it).

Comment (sjrd) [reply project/Build.scala]: I have looked at it, and it gets awfully more complicated ;)

File: project/Build.scala (hunk 1 above)
Comment (sjrd) [reply project/Build.scala]: Good catch! Removed.

File: project/Build.scala (hunk 2 above)
Comment (gzm0) [reply project/Build.scala]: That's was my hunch as well :)

**Pull request merged**